instruments/cryomagnetics/cryomagnetics_4g.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+"""
+Implements the cryomagnetics 4G superconducting magnet power supply
+"""
+
+# IMPORTS #####################################################################
+from __future__ import absolute_import
+
+import re
+from time import sleep
+
+import quantities as pq
+
+from instruments.abstract_instruments.cryomagnetics_instrument \
+    import CryomagneticsInstrument
+
+# CLASSES #####################################################################
+
+
+class Cryomagnetics4G(CryomagneticsInstrument):
+    """
+    Implements a Cryomagnetics 4G Superconducting magnet power supply.
```

Comment:
How do you propose to solve that with a generic Cryomagnetics Instrument? Should I use multiple inheritance?